LIReC/lib/manual_values/B_2.txt

```diff
@@ -0,0 +1,10 @@
+# A065421 (b-file synthesized from sequence entry)
```

Comment:
I could probably figure out an automatic way to download these from OEIS, then discard once we're done...

Reply:
Not sure I completely understand what these do, but in any case - I think generally they should be in source control iff you want to commit every time they change (and if they are some sort of cache of the LIReC then I think you don't)

Reply:
Well, as I mentioned these are taken directly from the online encyclopedia of integer sequences, and are mostly intended to be part of creating a new database. So, instead of having this massive folder, whenever necessary I can download constants from OEIS into the DB pretty much directly (up to parsing their format, but I got that covered already).

Reply:
I have worked in the past with a build system called Bazel which allows you to download artifacts during build time and use them in your code. Maybe we should consider uploading these files to an artifactory and integrating with a tool that allows us to download and use them on demand?

Reply:
Well, the only time we'd ever need these files is when creating a new DB, which I'd imagine would happen pretty rarely. Since `create_db.py` already does most of the work, I don't think there's much need to get fancy with it by using some special build tools or some such...